docs/developing/secrets_details.md

```diff
@@ -0,0 +1,40 @@
+---
+
+copyright:
+years: 2021
+lastupdated: "2021-08-16"
+
+---
+
+{:shortdesc: .shortdesc}
+{:new_window: target="_blank"}
+{:codeblock: .codeblock}
+{:pre: .pre}
+{:screen: .screen}
+{:tip: .tip}
+{:download: .download}
+
+# Secrets Manager details
```

Comment:
Can we say overview instead of details?

Reply:
The overview is in ./getting_started/overview_sm.md These are really details about how it works.